app/Filament/Server/Components/StatBlock.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\Filament\Server\Components;
+
+use Filament\Support\Concerns\Macroable;
+use Filament\Widgets\StatsOverviewWidget\Stat;
+use Illuminate\Contracts\Support\Htmlable;
+use Illuminate\Contracts\View\View;
+use Illuminate\Support\Str;
+use Illuminate\View\ComponentAttributeBag;
+
+class StatBlock extends Stat
+{
+    use Macroable;
```

Comment:
What does `Macroable` do?

Reply:
No idea, all I did the copy the Filament class and tweak it.